base-math/trunctdsf.c

```diff
@@ -63,15 +63,21 @@ CONVERT_WRAPPER(
 	          return SIGNBIT(a) ? -INFINITY : INFINITY;
 	      }
 	  }
-	else if (exp < -39)	/* Obvious underflow. */
+	else if (exp <= -BINPOWOF10_DENORM_FLT)	/* Obvious underflow. */
 	  {
 	    if (DFP_EXCEPTIONS_ENABLED)
 	      DFP_HANDLE_EXCEPTIONS (FE_UNDERFLOW|FE_INEXACT);
 
 	    switch (fegetround())
 	      {
 	        case FE_TONEAREST:
-              return SIGNBIT(a) ? -__FLT_DENORM_MIN__ : __FLT_DENORM_MIN__;
+	          mant = llabs(a_norm);
+	          if (exp < -BINPOWOF10_DENORM_FLT ||
+	                    ((exp == -BINPOWOF10_DENORM_FLT) &&
+	                     (mant >= 700649230000000)))
```

Comment:
Likewise for this number and indentation.

Reply:
Will be fixed.